tests/Validator/ErrorElementTest.php

```diff
@@ -120,7 +120,7 @@ public function testAddConstraint(): void
             $this->context->expects($this->once())
                 ->method('validateValue')
                 ->with($this->equalTo($this->subject), $this->equalTo($constraint), $this->equalTo(''), $this->equalTo('foo_core'))
-                ->will($this->returnValue(null));
+                ->willReturn(null);
```

Comment:
now that I think about it, isn't `null` like the default?

Reply:
like, can it work without this line.

Reply:
Nope, there is no default: `ArgumentCountError: Too few arguments to function PHPUnit\Framework\MockObject\Builder\InvocationMocker::willReturn(), 0 passed in /home/greg/dev/SonataAdminBundle/tests/Route/QueryStringBuilderTest.php on line 79 and exactly 1 expected`